BackofficeBundle/Form/DataTransformer/ReferenceToEmbedTransformer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelBundle\Manager\DocumentForEmbedManager;
+
+/**
+ * Class ReferenceToEmbedTransformer
+ */
+class ReferenceToEmbedTransformer implements DataTransformerInterface
+{
+    protected $documentForEmbedManager;
+    protected $formTypeName;
+
+    /**
+     * @param ObjectManager $objectManager
+     * @param string        $contentClass
+     */
+    public function __construct(DocumentForEmbedManager $documentForEmbedManager)
+    {
+        $this->documentForEmbedManager = $documentForEmbedManager;
+    }
+
+    /**
+     * @param string $documentName
+     */
+    public function setFormTypeName($formTypeName)
+    {
+        $this->formTypeName = $formTypeName;
+    }
+
+    /**
+     * Take a embed document array representation to return associative array formType id
+     *
+     * @param array $data
+     *
+     * @return array
+     */
+    public function transform($data)
+    {
+        if (!is_null($data)) {
+            return array($this->formTypeName => $this->documentForEmbedManager->fromDbToEntity($data)->getId());
```

Comment:
If you have multiple add in buildform, values will be sent with an associative array formTypeName => value, @nicolasThal